### PR TITLE
support releasing from TypeSpec

### PR DESCRIPTION
**Pull request by XiaofeiCao**
1. To release from TypeSpec, add `tspConfig` in `configure.yml`. If main branch, will use latest commit ID. If commit ID is provided, will use that commit ID:
e.g. `tspConfig: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/azurefleet/AzureFleet.Management/tspconfig.yaml`

2. `service` will be inferred from tspconfig.yaml's `service-dir`, to align with CI configuration
3. Test PR: https://github.com/Azure/azure-sdk-for-java/pull/40999

### CODE REVIEW

File: src/main/java/io/weidongxu/util/releaseautomation/LiteRelease.java [hunk 1]
```diff
@@ -90,67 +91,112 @@ public class LiteRelease {
...
-                    Optional<String> stableTag = readmeConfigure.getTagConfigures().stream()
+            prKeyword = sdk;
+            OUT.println("Releasing from TypeSpec, tsp-config file with commitID: \n" + tspConfig.getUrl());
+            OUT.println("sdk: " + sdk);
+            OUT.println("package-dir: " + tspConfig.getPackageDir());
+
+            variables.put("README", new Variable().withValue(sdk));
+            variables.put("TSP_CONFIG", new Variable().withValue(tspConfig.getUrl()));
+            templateParameters.put("RELEASE_TYPE", "TypeSpec");
+        } else { // generate from Swagger
```
Comment (XiaofeiCao) [inline src/main/java/io/weidongxu/util/releaseautomation/LiteRelease.java:121]: No major change on Swagger part.

File: src/main/java/io/weidongxu/util/releaseautomation/LiteRelease.java [hunk 2]
```diff
@@ -90,67 +91,112 @@ public class LiteRelease {
...
-            if (PREFER_STABLE_TAG) {
-                if (tag.endsWith("-preview")) {
-                    Optional<String> stableTag = readmeConfigure.getTagConfigures().stream()
+            prKeyword = sdk;
+            OUT.println("Releasing from TypeSpec, tsp-config file with commitID: \n" + tspConfig.getUrl());
+            OUT.println("sdk: " + sdk);
+            OUT.println("package-dir: " + tspConfig.getPackageDir());
+
+            variables.put("README", new Variable().withValue(sdk));
+            variables.put("TSP_CONFIG", new Variable().withValue(tspConfig.getUrl()));
```
Comment (XiaofeiCao) [inline src/main/java/io/weidongxu/util/releaseautomation/LiteRelease.java:119]: Use parsed url here, since `tsp-client` needs url that contains commitID.

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by weidongxu-microsoft** [comment]
Not sure whether the version in changelog would be correct or not (that probably belongs to the python script).

**Comment by XiaofeiCao** [comment]
> Not sure whether the version in changelog would be correct or not (that probably belongs to the python script).

I think it's python script. I'll check.

**Pull request merged**